addons/account/__init__.py

```diff
@@ -10,9 +10,14 @@
 
 SYSCOHADA_LIST = ['BJ', 'BF', 'CM', 'CF', 'KM', 'CG', 'CI', 'GA', 'GN', 'GW', 'GQ', 'ML', 'NE', 'CD', 'SN', 'TD', 'TG']
 
-def _auto_install_l10n(cr, registry):
+def _set_fiscal_country(env):
+    """ Sets the fiscal country on existing companies when installing the module.
+    """
```

Comment:
can you add a comment explaining why you need to call the compute yourself? blablabla orm limitation blablabla ?